apps/scripts/fib.c

```diff
@@ -0,0 +1,15 @@
+int fib(int n)
```

Comment:
Add header to the file.
- short description.
- Who is the owner of this test.
- when was this test created
- comment and Explain the memory regions in this test

Reply:
redundant. need to be removed